src/services/getCourtCaseByOrganisationUnit.ts

```diff
@@ -27,7 +27,7 @@ const getCourtCaseByOrganisationUnit = (
   leftJoinAndSelectTriggersQuery(query, user.excludedTriggers)
     .leftJoinAndSelect("courtCase.notes", "note")
     .leftJoin("note.user", "user")
-    .addSelect(["user.forenames", "user.surname"])
+    .addSelect(["user.forenames", "user.surname", "user.visibleForces", "user.username"])
```

Comment:
Does this add any slow downs in the other various places this gets used? E.g. Case List index page?